src/pages/schedule.js

```diff
@@ -0,0 +1,365 @@
+import React from 'react';
+const moment = require('moment-timezone');
+import Announcement from '../components/announcement';
+
+import './schedule.css';
+
+export default class Schedule extends React.Component {
+  state = {}
+
+  componentDidMount() {
+    this.updateHandle = setInterval(this.updateTime, 1000 * 60)
+  }
+
+  componentWillUnmount() {
+    clearInterval(this.updateHandle);
+  }
+
+  updateTime = () => {
+    this.setState({
+      currentTime: nowInUTC()
+    });
+  }
+
+  renderDay = (day) => {
+    const currentTime = this.state.currentTime || nowInUTC();
+    return (
+      <div className="schedule__day">
+        <h2 className="schedule__headlineDay">{day.title}</h2>
+        <div className="schedule__timeTable">
+          {day.entries.map((entry) => {
+            const isCurrent = currentTime >= entry.startTime && currentTime <= entry.endTime;
+            const entryStart = utcTSToBerlinTZ(entry.startTime);
+            const entryEnd = utcTSToBerlinTZ(entry.endTime);
+
+            const time = "" +
```

Comment:
Might be slightky nicer with es6 templating

Reply:
Ah yes, good catch